src/ResourceManager/Sql/Commands.Sql/Services/Util.cs

```diff
@@ -89,7 +89,7 @@ public static bool AreEmailAddressesInCorrectFormat(string emailAddresses, char
 
             string[] emailAddressesArray = emailAddresses.Split(seperator).Where(s => !string.IsNullOrEmpty(s)).ToArray();
             var emailRegex =
-                new Regex(string.Format("{0}{1}",
+                new Regex(string.Format("(?i){0}{1}",
                     @"^(?("")("".+?(?<!\\)""@)|(([0-9a-z]((\.(?!\.))|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w])*)(?<=[0-9a-z])@))",
                     @"(?(\[)(\[(\d{1,3}\.){3}\d{1,3}\])|(([0-9a-z][-\w]*[0-9a-z]*\.)+[a-z0-9][\-a-z0-9]{0,22}[a-z0-9]))$"));
             return !emailAddressesArray.Any(e => !emailRegex.IsMatch(e));
```

Comment:
The warning should tell the user which specific email address is considered invalid.

Reply:
Hi,

We also have a validation in the server-side
I guess this exists in order to prevent rest APIs calls which Definitely going to fail.
I am fixing the error message.

Reply:
If there is validation server-side, then I don't think client-side is also needed.

Reply:
ok. I will remove this check